src/components/Loading/Loading.js

```diff
@@ -0,0 +1,18 @@
+import PulseLoader from 'react-spinners/PulseLoader'
+import FadeLoader from 'react-spinners/FadeLoader'
+import { css } from "@emotion/react";
+
+const Loading = ({loading, type}) => {
+
+  const override = css`
+    display: block;
+    margin: 0 auto;
+    color: red;
+  `;
+
+  return (!type)
```

Comment:
This logic for the CSS makes sense, but would you mind going over it together before we merge?